doc/help/kdb-record.md

```diff
@@ -0,0 +1,144 @@
+If it is the currently active session, it will be stopped.
+
+### `mark [--meta meta_name] <key_name>`
+This command marks a key to be used for validation when importing the recording.
+By default, the values of a marked key will be compared when importing.
+The import will fail if the key has a different value.
+
+Using the `--meta` flag, one can specify to use the value of a specific meta instead of the value of the key.
+
+### `list`
+Lists all known recording sessions.
+
+### `name`
+Prints out the name of the currently active recording session.
+If no session is active, the return code is `1`, otherwise `0`.
+
+### `changes [session_name]`
+Shows all the changes in the specified session in human-readable format.
+Outputs to `stdout`.
+If no session is specified, the currently active session is used.
+
+### `export  [--replay|--diff] [session_name]`
+Exports the changes made in the specified session.
+Outputs to `stdout`.
+If no session is specified, the currently active session is used.
+
+The `export` command has the ability to output two different types: diffs and replays.
+
+The `--diff` flag (default) only exports what has actually changed in the recording session.
+This means, e.g., that when you add a key and remove it in the same session, it will not be exported.
+
+The `--replay` flag creates a file that contains every modifying operation made in the session in the order they were performed.
+This is regardless whether at the end of the session the value has really changed or is the same as before.
+Files created using `--replay` serve as a transaction log.
+
+
+### `import [--disable-validation] <file_name>`
+Imports files created by the `export` command.
+
+By default we check whether the current values in the key database match the "old" values in the files.
+The values of marked keys will also be checked.
+Those checks can be disabled using the `--disable-validation` flag.
+
+
+## EXAMPLE
+
+```
+$ kdb record start AddNewHosts
+AddNewHosts
+$ kdb record mark system:/hosts/ipv6/localhost
+Marked system:/hosts/ipv6/localhost to be ::1
+$ kdb set system:/hosts/ipv4/www.example.com 8.8.8.8
+$ kdb set system:/hosts/ipv4/www.beispiel.de 4.4.4.4
+$ kdb rm system:/hosts/ipv4/www.ejemplo.es
+$ kdb record changes
+Marked system:/hosts/ipv6/localhost to be ::1
+Changed system:/hosts/ipv4/www.example.com from 1.2.3.4 to 8.8.8.8
+Added system:/hosts/ipv4/www.beispiel.de with value 4.4.4.4
+Removed system:/hosts/ipv4/www.ejemplo.es
+$ kdb record export --replay
+<export output. format not yet defined>
+$ kdb record list
+AddNewHosts
+SomeOtherSession
+$ kdb record name
+AddNewHosts
+$ ELEKTRA_RECORD_ACTIVE_SESSION=JustATest kdb record name
+JustATest
+$ kdb record reset
+$ kdb record changes
+<no changes>
+$ kdb record import /tmp/invalid
+Error importing! 
+Existing value for key system:/hosts/ipv6/localhost does not match! (shoud be ::1, is ::2)
+$ kdb record import --disable-validation /tmp/invalid
+Successfully imported!
```

Comment:
I agree this example is not clear enough.

If `system:/hosts/ipv6/localhost` was only a marked key, its value has not changed. Otherwise it has the value from the imported file.

All other keys have the value from the imported file.